### PR TITLE
This closes #125

### LINKED ISSUE #125: Button stacking leads to display issues on hover event
@morebrownies another border issue

image

and here:

image

both message and follow have the issue

### PR DESCRIPTION
**Pull request by jjeffryes**
This will update the storeStyles branch to only remove the right border on buttons in a button strip when they aren't in a hover state.

### CODE REVIEW

File: styles/_variables.scss [hunk 1]
```diff
@@ -13,7 +13,7 @@ $focusedBorder: adjust-color($text, $lightness: 35%);
 $emphasis1: rgb(253,166,57);
 $emphasis2: rgb(174, 138, 88);
 $error: red;
-$overlay: adjust-color($secondary, $lightness: 0%, $alpha: -0.1);
+$overlay: adjust-color($secondary, $alpha: -0.1);
```
Comment (jjeffryes) [inline styles/_variables.scss]: I accidentally included this fix to one of the comments I made, so free bonus.

File: styles/components/_buttons.scss [hunk 2]
```diff
@@ -21,12 +21,13 @@
...
     text-decoration: none;
     cursor: pointer;
-    border: solid 1px $focusedBorder;
+    border-color: $focusedBorder;
   }
 
   &:focus {
     text-decoration: none;
     outline: 0;
+    border-color: $focusedBorder;
```
Comment (jjeffryes) [inline styles/components/_buttons.scss]: The border should change on focus too, for anyone using a keyboard to tab to the buttons.

**Comment by rmisio** [comment]
This ends up causing the button to jump by a pixel on hover.

@morebrownies @jjeffryes Is that ok?

**Comment by morebrownies** [comment]
It would prefer if the button didn't jump a pixel.

**Comment by jjeffryes** [comment]
Good catch. It shouldn't do that. I'll fix it.

**Comment by rmisio** [comment]
Can we omit the hover border on disabled elements?

image

**Comment by jjeffryes** [comment]
The code is updated to not affect disabled elements, or children of disabled elements. It also is moved to the themes file so it can add a different focus color for different border classes if desired.

Note that this does not affect the screenshot, as that element has a span inside it with an opacity set, rather than being disabled. We should change the view code to add a disabled class to that button when it's not usable.

**Comment by hoffmabc** [comment]
@jjeffryes do you mind leaving a message when closing for historical purposes to understand what happened?

**Comment by jjeffryes** [comment]
My bad there. This is a branch from a PR that was closed, so it's no longer a viable PR.

**Pull request closed without merging**